test/gtest/uct/test_event.cc

```diff
@@ -166,20 +170,38 @@ void test_uct_event::test_recv_am(unsigned arm_flags, unsigned send_flags)
     free(recv_buffer);
 }
 
-UCS_TEST_SKIP_COND_P(test_uct_event, am,
+/*
+ * Disable flow control (if transport supports it), and expect at most 2%
+ * spurious events.
+ */
+UCS_TEST_SKIP_COND_P(test_uct_event, am_no_fc,
                      !check_caps(UCT_IFACE_FLAG_CB_SYNC |
                                  UCT_IFACE_FLAG_AM_BCOPY) ||
-                     !check_event_caps(UCT_IFACE_FLAG_EVENT_RECV))
+                     !check_event_caps(UCT_IFACE_FLAG_EVENT_RECV),
+                     "RC_FC_ENABLE?=n")
 {
-    test_recv_am(UCT_EVENT_RECV, 0);
+    test_recv_am(UCT_EVENT_RECV, 0, 0.02);
+}
+
+/*
+ * Enable flow control (skip the test if transport does not support it), and
+ * expect at most 10% spurious  events.
```

Comment:
extra whitespace before `events`

Reply:
removed the comment, not relevant anymore